.github/workflows/upgrade-one-python-dependency.yml

```
@@ -12,6 +12,10 @@ on:
          description: 'Name of package to upgrade'
          required: true
          type: string
+       version:
+          description: 'Version number to upgrade to - required if version is pinned'
```

[nit] Suggested rewording for clarity:
```suggestion
          description: 'Version number to upgrade to in constraints.txt (only needed if pinned)'
```